internal/errors.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.21.0
-// 	protoc        v3.10.1
+// 	protoc        v3.7.0
```

Comment:
I can remove this, or commit it from all the other `.pb.go` files where the bazel run has changed just this line. What do you think?

Reply:
I think we should be able to remove this

Reply:
(How) do we control the protoc version used?

Reply:
It's in the build environment [Dockerfile](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/.circleci/Dockerfile#L20) at the moment. Clearly this file isn't being regenerated.